### PR TITLE
1 Scanner, 1 Computer, X Clonepods

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl: Cyberboss
add: You can now link additional cloning pods in the same powered area to a single computer using a multitool.
/:cl:

Also fixes some failing GCs between the clonepod and cloneputer

### CODE REVIEW

**Comment by coiax** [comment]
but I was working on this in my cloning overhaul

**Comment by Cyberboss** [comment]
I am an emotional coder, and waiting toys with my emotions.

That, and the shitcode that is /tg/

**Comment by coiax** [comment]
I was going to have cloning networks be built either out of adjacent machines or with wires connecting them like network cables.

**Comment by Cyberboss** [comment]
@KorPhaeron comment?

**Comment by XDTM** [comment]
I like multitool linking more than cables tbh, it's also consistent with other machine linking

**Comment by lzimann** [comment]
this is possibly better than having to deal with wires/build them adjacent

**Comment by RandomMarine** [comment]
Multitool linking is more consistent and convenient.

**Comment by coiax** [comment]
yay for merge conflicts and procrastinating

**Comment by Anonmare** [comment]
Does this mean I can have clone farms?

**Comment by SgtxRootbeer** [comment]
Does it have to be in the same powered area? I kinda like having a scanner near the morgue, but pods near cryo.

**Comment by Cyberboss** [comment]
Yes. Medbay is all 1 area I believe so you should be good

**Comment by MrStonedOne** [comment]
See, I don't like this mechanic, too hand wavy with the IC part, and the assumption has been the 3 parts are physically connected.

Make not use mutitools and areas, (a shitty mechanic lore wise) but require it be cardinally adjacent. this 1: limits it to 1 additional pod, so it doesn't throw out all balance around clone times because lel 500 clone pods, and two, doesn't throw out immersions with more hand wavy "wireless" bullshit.

**Comment by XDTM** [comment]
If you can make an extra cloner you can make several already, they're not very pricey, it's just that nobody bothers. I agree it should be range based instead of area based, but wireless connection is not hand wavy bullshit since all you're doing is transferring genetic data, not matter.

**Comment by XDTM** [comment]
And cloning times are already extremely variable depending on if the doctors are speedcloning or not and if the cloner is upgraded. Two or more pods at the same time mean that you can't just shut the power to cloning to eject without also ejecting every other clone.

File: code/game/machinery/computer/cloning.dm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	circuit = /obj/item/weapon/circuitboard/computer/cloning
 	req_access = list(access_heads) //Only used for record deletion right now.
 	var/obj/machinery/dna_scannernew/scanner = null //Linked scanner. For scanning.
-	var/obj/machinery/clonepod/pod1 = null //Linked cloning pod.
+	var/list/pods = list() //Linked cloning pods
```
Comment (RemieRichards) [inline code/game/machinery/computer/cloning.dm]: lazyinit this

**Comment by RandomMarine** [comment]
If someone wants to take the time to fill a room with 20 clone pods, they should be able to. 
They should also expect an antag to blow it all up. And if there's no antag to blow them up, then most of those clone pods are just going to collect dust.

The sprites don't even look like they're physically connected, so I'm not sure where immulsions play into this.

Also, I believe this paves the way to making cutting off power to the clone systems make !BAD THINGS! happen instead of !GOOD THINGS!, like it should.

**Comment by Cyberboss** [comment]
@RandomMarine Along with @coiax's other upcoming cloning changes you should get exactly what you ask for

**Comment by KorPhaeron** [comment]
Pretty cool, won't fit in most default medbay maps though

**Comment by KorPhaeron** [comment]
Pretty cool

**Comment by ExcessiveUseOfCobblestone** [comment]
If you think this needs balance, why not add an exponential power increase on the computer managing the clone pods? I mean it's not much because > infinite power but if you cut a wire to medbay you can shut anti-murderbone factories out pretty rapidly.

**Comment by Cyberboss** [comment]
Great idea

**Comment by optimumtact** [comment]
Merging this will give coaix even more incentive to finish his rework

**Pull request merged**